### PR TITLE
domain manager crashes when dom-u virtual mode is changed from PV to HVM or vice versa

### PR DESCRIPTION
**Pull request by spatchineelam**
When domain config is modifed, it is inactivated and later activated.
During inactivation UUID is set to NIL, but is not being set on activation
for this case.
Invoking configAdapters() even in domain config modification.

### CODE REVIEW

File: pkg/pillar/cmd/domainmgr/domainmgr.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/lf-edge/eve/pkg/pillar/sema"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/lf-edge/eve/pkg/pillar/wrap"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```
Comment (eriknordmark) [inline pkg/pillar/cmd/domainmgr/domainmgr.go]: Why change this? Not needed and might make others think we need the same or similar changes in other files.

Comment (spatchineelam) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: I haven't modified this, got changed automatically when built.
The next line of code on line 40 has similar change
log "github.com/sirupsen/logrus"

I will remove it manually

**Comment by spatchineelam** [comment]
Removed uuid from 
uuid "github.com/satori/go.uuid"

File: pkg/pillar/cmd/domainmgr/domainmgr.go (hunk 1 above)
Comment (eriknordmark) [reply pkg/pillar/cmd/domainmgr/domainmgr.go]: FWIW The line for logrus has always been that way with "log" up front.

**Comment by eriknordmark** [comment]
> Removed uuid from
> uuid "github.com/satori/go.uuid"

I don't see that removal. Did you push?

**Comment by spatchineelam** [comment]
When domain config is modifed, it is inactivated and later activated.
During inactivation UUID is set to NIL, but is not being set on activation
for this case.
Invoking configAdapters() even in domain config modification.

**Comment by spatchineelam** [comment]
I have installed some vim plugins for golang development, and these have been causing issues.
Had to reset all the unnecessary commits and merges tot his branch, and redo the commit.

**Comment by spatchineelam** [comment]
I have installed some vim plugins for golang development, and these have been causing issues.
Had to reset all the unnecessary commits and merges tot his branch, and redo the commit.

**Comment by spatchineelam** [comment]
> > Removed uuid from
> > uuid "github.com/satori/go.uuid"
> 
> I don't see that removal. Did you push?

Yes, I have pushed the fix.

**Comment by eriknordmark** [comment]
Have you verified whether this also fixes the other crashes with the same stack trace?

**Comment by spatchineelam** [comment]
I am midway thru verifying.

**Pull request merged**